.github/workflows/packages.yml

```diff
@@ -28,7 +28,8 @@ jobs:
 
       - name: Patching pygeodiff binaries
         run: |
-          echo "no-op !! Here is the place to patch linkage !!"
+          patchelf --replace-needed libsqlite3-d9e27dab.so.0.8.6 libsqlite3.so.0.8.6 ./pygeodiff-binaries/libpygeodiff-1.0.0-python.so
```

Comment:
1. `d9e27dab` is most probably name-mangling addition of audit wheel command. This shouldn't be hardcoded here, but rather ldd + regex should be used to get the mangled name + the version of sqlite (0.8.6) and this should be used instead
2. don't you want to link agains libsqlite3.so (without version suffix?)
3. libpygeodiff-1.0.0 version shouldn't be hardcoded, use libpygeodiff-${GEODIFF_VER}